cli/testdata/bpmetadata/content/examples-some-without-tf/examples/acm/acm-terraform-blog-part1/README.md

```diff
@@ -0,0 +1,76 @@
+# Enable ACM features with Terraform - Part 1
```

Comment:
Since this test data is for dir testing purposes, lets remove file contents in all of `contest/*` files and replace with minimal mock content.

Reply:
Condensed as much as I could.